packages/react-server/src/ReactFlightServer.js

```diff
@@ -2299,6 +2301,25 @@ function visitAsyncNode(
           // Undefined is used as a signal that we found a suitable aborted node and we don't have to find
           // further aborted nodes.
           return undefined;
+        }
+        if (request.status === ABORTING) {
+          if (node.start < request.abortTime && node.end > request.abortTime) {
+            // We aborted this render. If this Promise spanned the abort time it was probably the
+            // Promise that was aborted. This won't necessarily have I/O associated with it but
+            // it's a point of interest.
+            // However, if the Promise and IO node have the same owner, it likely means a sync component
+            // created both the Promise and initiated the I/O. Prefer the IO node for more specific info.
```

Comment:
This part doesn't make sense to me. Why would this be an indicator that it's better?

Reply:
Whoops, that's a leftover I forgot to remove. We actually don't need this condition.